src/common/gfx/gfxSDL.cpp

```diff
@@ -196,6 +203,12 @@ void GraphicsSDL::FlipScreen()
 {
     SDL_UpdateTexture(sdl2_screen_as_texture, NULL, screen->pixels, screen->pitch);
     SDL_RenderClear(sdl2_renderer);
+#ifdef __SWITCH__
+    // split/combine joycons depending on singleJoyconMode and handheld/docked mode
+    update_joycon_mode();
+    // turn on linear filtering to make all pixels equal square-shapes, and a bit blurry)
+    SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "linear");
```

Comment:
You should rather replace the original SDL_SetHint already in place, at L156

Reply:
Also to reiterate what I stated in the issue comments, it should be a setting, so that it fits all tastes, I plan to add a "nearest integer scale" mode which would be smaller but would look nice in "nearest" mode.

Reply:
Yes I agree but I need to find a way to compile it first. At the moment when I run your build.sh it says it cannot find libnx. It seems like it is looking in /use/local, not /opt/devkitpro